test/tools/sh_do_test.sh

```diff
@@ -8,13 +8,11 @@ if [ "${RESULTS_DIR}" == "" ]; then
     exit 1
 fi
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Why did you remove this?
This allows the script to be called from anywhere (in the future at least)

Reply:
I think this might be why AppVeyor is failing...I"ve removed it to see if it fixes the issue... I know removing it is more "brittle".  If this works, maybe I should redefined it to something like `${TEST_DIR}/../tools`?  I don't like it as much.  The fact that Bash script files don't have an easy way to get their own directory is such a pain.  This is one thing that BATCH has over Bash :) (`%~dp0`)

Reply:
Adding `bash` to the beginning of the `spawnShell` command seems to have fixed the AppVeyor issue.